envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/listeners/filters/RBACFilterFactoryTest.kt

```diff
@@ -90,6 +90,32 @@ internal class RBACFilterFactoryTest {
         assertThat(generated).isEqualTo(rbacBuilder)
     }
 
+    @Test
+    fun `should generate RBAC with different rules for incoming permissions`() {
```

Comment:
Thanks for migrating this test from EnvoyIngressRoutesFactoryTest. Now we have complete set of tests.
 I think we should also remove that incoming permissions from the source test, as they are not relevant now (the test: "should create route config with two simple endpoints and response timeout defined"). Or even remove the whole test if it makes no sense now (for sure we have to rename it at least)

Reply:
I think it's worthy to check that admin routes are generated. I will rename it.